cloudify_handler/cloudify.py

```diff
@@ -29,10 +35,21 @@ def _bind(self):
            to set auto_delete=True)
         """
         credentials = pika.PlainCredentials(self.user, self.password)
+
+        ssl_enabled = broker_config.broker_ssl_enabled
+
+        broker_port, ssl_options = utils._get_broker_ssl_and_port(
```

Comment:
i don't like that you are using a "private" function here. If you think this is internal and should not be exposed as public user api, then put in Internal, similarly to what happens here
https://github.com/cloudify-cosmo/cloudify-plugins-common/pull/142/files#diff-a28c7ed4f8758a0a866f5b664e6ca4f4R220